Userland/Applications/TextEditor/MainWidget.h

```diff
@@ -53,6 +74,8 @@ class MainWidget final : public GUI::Widget {
 
     virtual void drop_event(GUI::DropEvent&) override;
 
+    RefPtr<FileSystemAccessClient> m_filesystem_client;
```

Comment:
According to the comment on the constructor:
```suggestion
    NonnullRefPtr<FileSystemAccessClient> m_filesystem_client;
```